src/DependencyInjection/Providers/TwitchHelixProviderConfigurator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * OAuth2 Client Bundle
+ * Copyright (c) KnpUniversity <http://knpuniversity.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @Author Erdem Uyanik
+ */
+
+namespace KnpU\OAuth2ClientBundle\DependencyInjection\Providers;
+
+use Symfony\Component\Config\Definition\Builder\NodeBuilder;
+
+class TwitchHelixProviderConfigurator implements ProviderConfiguratorInterface
+{
+    public function buildConfiguration(NodeBuilder $node)
+    {
+        // no custom options
+    }
+
+    public function getProviderClass(array $config)
+    {
+        return 'Vertisan\OAuth2\Client\Provider\TwitchHelix';
+    }
+
+    public function getProviderOptions(array $config)
+    {
+        return [
+            'clientId' => $config['client_id'],
+            'clientSecret' => $config['client_secret'],
+        ];
+    }
+
+    public function getPackagistName()
+    {
+        return 'vertisan/oauth2-twitch-helix';
+    }
+
+    public function getLibraryHomepage()
+    {
+        return 'https://github.com/vertisan/oauth2-twitch-helix';
+    }
+
+    public function getProviderDisplayName()
+    {
+        return 'Twitch';
```

Comment:
Hm, probably `Twitch Helix` to be different from `TwitchProviderConfigurator`'s name?